yt/utilities/command_line.py

```diff
@@ -1,3 +1,4 @@
+# isort: skip-file
```

Comment:
note that it'd be easier to manage if instead of this comment, the file was added to isort's ignore list in setup.cfg, but it shouldn't hold this PR back.

@matthewturk , can you do it in https://github.com/yt-project/yt/pull/2756 ?